### ISSUE TITLE
diffie-hellmann: unused variable in test

### ISSUE THREAD

**Issue by coriolinus** [exercism/rust#404]
```shell
diffie-hellman$ cargo test
   Compiling diffie-hellman v0.1.0 (file:///mnt/d/Users/coriolinus/Documents/Projects/exercism.io/rust/diffie-hellman)
warning: unused variable: `g`
  --> tests/diffie-hellman.rs:31:9
   |
31 |     let g: u64 = 7;
   |         ^
   |
   = note: #[warn(unused_variables)] on by default
   = note: to avoid this warning, consider using `_g` instead

    Finished dev [unoptimized + debuginfo] target(s) in 2.62 secs
     Running target/debug/deps/diffie_hellman-6af69c79bf0abef3

running 1 test
test modular_math::tests::test_mul_mod ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/diffie_hellman-5c71c246f13515e9

running 4 tests
test test_public_key_correct ... ok
test test_secret_key_correct ... ok
test test_changed_secret_key ... ok
test test_private_key_in_range_key ... ok

test result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

   Doc-tests diffie-hellman

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```

Let's get rid of that unused `g`.
